sdk/provisioning/Azure.Provisioning.AppService/tests/BasicAppServiceTests.cs

```diff
@@ -81,12 +82,12 @@ await test.Define(
                                     new AppServiceNameValuePair
                                     {
                                         Name = "AzureWebJobsStorage",
-                                        Value = BicepFunction.Interpolate($"DefaultEndpointsProtocol=https;AccountName={storage.Name};EndpointSuffix=core.windows.net;AccountKey={storage.GetKeys()[0].Value!.Value}")
+                                        Value = BicepFunction.Interpolate($"DefaultEndpointsProtocol=https;AccountName={storage.Name};EndpointSuffix=core.windows.net;AccountKey={storage.GetKeys()[0].Unwrap().Value}")
```

Comment:
This is an example of where things break down a little.  When you have a collection of models, things still get wrapped in a `BicepValue<T>` (i.e., `public BicepList<StorageAccountKey> GetKeys()` is the signature which means `GetKeys()[0]` is a `BicepValue<StorageAccountKey>`).  `Unwrap` turns `BicepValue<StorageAccountKey>` into an identical `StorageAccountKey` where everything's easier to access again.  These aren't super common, so this tradeoff was less awful than duplicating collections for models.